ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.less

```diff
@@ -62,14 +59,18 @@
 
 @{customElementNamespace}filter.is-location {
   > .filter-input {
-    margin-top: @minimumSpacing;
     display: block;
+    padding: @minimumSpacing 1.5rem 0px @minimumSpacing + 0.75 *
+      @minimumButtonSize + @minimumButtonSize;
+    min-width: 19 * @minimumFontSize;
+    margin: 0px;
+    height: auto;
```

Comment:
Do you need both this line and the logic associated with this one: https://github.com/codice/ddf/pull/5118/files#diff-86ab3c1a2398777fe2ed057722ea4367R115? Seems like it's doing the same thing but CSS is strange sometimes

Reply:
Yes technically the `auto` on L116 of location.js is necessary only for the purposes of correct syntax in the ternary expression. It could not stand alone without this setting to accomplish the same goal